syntax = "proto3";

package conversation_service;

import "google/protobuf/empty.proto";

option go_package = "github.com/milkyhoop/template-service-golang/internal/delivery/pb/template;template";
// option python_package = "app.protos";

service Conversation_service {
  rpc DoSomething (Conversation_serviceRequest) returns (Conversation_serviceResponse);
  rpc HealthCheck (google.protobuf.Empty) returns (google.protobuf.Empty);
}

message Conversation_serviceRequest {
  string user_id = 1;
  string input = 2;
}

message Conversation_serviceResponse {
  string status = 1;
  string result = 2;
}
